js/models/LocalSettings.js

```diff
@@ -0,0 +1,36 @@
+import LocalStorageSync from '../utils/backboneLocalStorage';
```

Comment:
Made a model that syncs via LocalStorage to contain our local settings. Rather than directly going into LocalStorage, this gives us a centralized API that relevant areas of the app can read / write from. It also gives us a bunch of model sugar, like a place to centralize validation, change events, etc...